DigitalLearningSolutions.Data/DataServices/SupervisorService.cs

```diff
@@ -442,7 +442,7 @@ SELECT MAX(sar2.DateTime)
 						    WHERE sar2.SelfAssessmentID = sar.SelfAssessmentID AND sar2.CompetencyID = co.ID
 					)
                 WHERE (sd.SupervisorAdminID = @adminId) AND (cas.Removed IS NULL) AND (sasv.Verified IS NULL) AND (sd.Removed IS NULL)
-				GROUP BY sa.ID, ca.ID, sd.ID, u.FirstName, u.LastName, sa.Name", new { adminId }
+				GROUP BY sa.ID, ca.ID, sd.ID, u.FirstName, u.LastName, sa.Name,sasv.Requested", new { adminId }

```

Comment:
Think we need to group on Date (Day) rather than DateTime for the interface to make sense.

Reply:
Discussed with Madhavi on this and explained the scenario. Confirmation link will take the user to display all requests. So no need to show all records which looks duplicate. So we will be grouping with Date which will help to identify on which date the request is being created.
![image](https://user-images.githubusercontent.com/120369940/218461550-572a6c3f-7e21-47be-9ac6-6667236d05d0.png)